.github/workflows/postgresql.yml

```
@@ -0,0 +1,118 @@
+      OS: ${{ matrix.os }}
+      PYTHON: ${{ matrix.python-version }}
+      CRATEDB_VERSION: ${{ matrix.cratedb-version }}
+      POSTGRESQL_VERSION: ${{ matrix.postgresql-version }}
+      UV_SYSTEM_PYTHON: true
+      TC_KEEPALIVE: true  # Do not tear down Testcontainers
+
+    services:
+      cratedb:
+        image: crate/crate:${{ matrix.cratedb-version }}
+        ports:
+          - 4200:4200
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)